storage/innobase/xtrabackup/src/redo_log.cc

```diff
@@ -1177,26 +1177,16 @@ bool Redo_Log_Data_Manager::is_error() const { return (error); }
 Redo_Log_Data_Manager::~Redo_Log_Data_Manager() { os_event_destroy(event); }
 
 bool Redo_Log_Data_Manager::stop_at(lsn_t lsn) {
-  bool last_checkpoint = reader.find_last_checkpoint_lsn(&last_checkpoint_lsn);
-  if (last_checkpoint) {
-    msg("xtrabackup: The latest check point (for incremental): '" LSN_PF "'\n",
-        last_checkpoint_lsn);
-  }
   msg("xtrabackup: Stopping log copying thread at LSN " LSN_PF ".\n", lsn);
-
   stop_lsn = lsn;
   os_event_set(event);
   thread.join();
 
   archived_log_monitor.stop();
-
-  scanned_lsn = reader.get_scanned_lsn();
-
-  if (last_checkpoint_lsn > scanned_lsn) {
-    msg("xtrabackup: error: last checkpoint LSN (" LSN_PF
-        ") is larger than last copied LSN (" LSN_PF ").\n",
-        last_checkpoint_lsn, scanned_lsn);
-    return (false);
+  bool last_checkpoint = reader.find_last_checkpoint_lsn(&last_checkpoint_lsn);
+  if (last_checkpoint) {
+    msg("xtrabackup: The checkpoint LSN in redo log file  '" LSN_PF "'\n",
+        last_checkpoint_lsn);
   }
```

Comment:
why this check is removed?

Reply:
readded with lsn instead of last_checkpoint.

Reply:
readded the check.